### ISSUE TITLE
Feature for msig to copy (replicate) existing proposal

### ISSUE THREAD

**Issue by coachbjork** [coachbjork/alienw_wallet#20]
The idea is the same as copy msig on bloks or waxblock.

by hitting that button, you get into an edit msig screen, where you can redo the exact same msig, or change stuff in it before you publish it.

E.g.
- msig for transfer of 10k TLM from DAO to account1, account2, account3.
- Hit copy button, gets a screen with the same msig pre-filled, where the user can change it to e.g. 15k TLM for account1, account2 and a new account4.

So basically a speeded up, easy way to replicate a transaction, but given option to edit the content, remove something, change, or add another action.
